tests/ValidateSbrpTests/ValidateSbrp.Tests.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(NetCurrent)</TargetFramework>
+    <Nullable>enable</Nullable>
+    <NoWarn>$(NoWarn);CS8602</NoWarn>
```

Comment:
I don't think these nullable related warnings should be suppressed rather they should be addressed in code.

FYI, you could define these as one property - <NoWarn>$(NoWarn);CS8602;CS8603</NoWarn>

Reply:
Removed in https://github.com/dotnet/source-build-reference-packages/commit/ce1a70dc071c6a5adce2b2d0ca6285031ee12258 and added null-forgiving operator to relevant lines in `DummyAttributeTypeProvider.cs`